src/context.rs

```diff
@@ -19,7 +20,7 @@ use crate::{Actor, Address, Handler, KeepRunning};
 
 /// `Context` is used to control how the actor is managed and to get the actor's address from inside
 /// of a message handler.
-pub struct Context<A> {
+pub struct Context<A, TStage = Running> {
```

Comment:
Currently setting this as the default to reduce the diff. We may want to be explicit about it in the final version.

Reply:
Sounds sensible - what does the T in TStage stand for exactly, though?

Reply:
It stands for "Type" and is a convention I sometimes use for type parameters that have more descriptive names.

I don't feel strongly about it though, happy to rename it to whatever :)